torchvision/prototype/models/detection/mask_rcnn.py

```diff
@@ -28,6 +28,9 @@ class MaskRCNN_ResNet50_FPN_Weights(WeightsEnum):
         url="https://download.pytorch.org/models/maskrcnn_resnet50_fpn_coco-bf2d0c1e.pth",
         transforms=CocoEval,
         meta={
+            "task": "image_object_detection",
+            "architecture": "FasterRCNN",
```

Comment:
I would say the `architecture` for this one should probably be `MaskRCNN`, as it's an extension of Faster R-CNN which has its own paper https://paperswithcode.com/paper/mask-r-cnn